packages/instrumentation-aws-sdk/src/utils.ts

```diff
@@ -14,9 +14,10 @@ export const removeSuffixFromStringIfExists = (str: string, suffixToRemove: stri
 
 export const normalizeV2Request = (awsV2Request: Request<any, any>): NormalizedRequest => {
     const service = (awsV2Request as any)?.service;
+    const operation = (awsV2Request as any).operation;
     return {
-        serviceName: service?.serviceIdentifier?.toLowerCase(),
-        commandName: toCamelCase((awsV2Request as any).operation),
+        serviceName: service?.api?.serviceId,
+        commandName: toPascalCase(operation),
```

Comment:
I couldn't find the right casing anywhere in the API, (in fact it looks like it was already in CamelCase casing everytime?) so I modified the helper to be `toPascalCase` which is otherwise known as "Upper Camel Case" or "Camel Case with the first letter uppercase".

Reply:
Yeah,
Wander what is the motivation for it to be camelCase. Can see that in the [apis JSONs](https://github.com/aws/aws-sdk-js/blob/master/apis/dynamodb-2012-08-10.min.json) it's in the right PascalCase